stdlib/database/h2/src/main/java/org/ballerinalang/database/h2/CreateClient.java

```diff
@@ -52,7 +52,7 @@ public void execute(Context context) {
         Struct clientEndpointConfig = BLangConnectorSPIUtil.toStruct(configBStruct);
         Map<String, Value> dbOptions = clientEndpointConfig.getMapField(Constants.EndpointConfig.DB_OPTIONS);
         String urlOptions = "";
-        if (dbOptions != null) {
+        if (!dbOptions.isEmpty()) {
```

Comment:
Can't this become null ? 
[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-core/src/main/java/org/ballerinalang/connector/impl/StructImpl.java#L129

Reply:
Even if we don't pass any dbOptions, an empty map will be present. I have checked at the Line 129 but seems an empty map is passed there too.